src/desktop/components/inquiry_questionnaire/views/account.coffee

```diff
@@ -57,13 +57,31 @@ module.exports = class Account extends StepView
       @submit(form, recaptcha_token)
     )
 
+  handle2FAError: (form, response) ->
+    errorMessage = response.responseJSON.error
+    if errorMessage is "missing two-factor authentication code"
+      # reset form state
+      form.state.call(form, null)
+      form.reenable.call(form)
+      # reveal OTP field
+      form.$form.find('.iq-otp-field').removeClass('is-hidden')
+      wasHandled = true
+    else if errorMessage is "invalid two-factor authentication code"
+      # display error
+      form.error.call(form, errorMessage)
+      wasHandled = true
```

Comment:
We could have omitted this branch, and used the default error handling — but would have ended up with a generic "There was an error" message rather than informing the user that their code was invalid. This seemed marginally better to me.